src/build.d

```diff
@@ -513,6 +514,25 @@ alias checkwhitespace = memoize!(function()
     return new DependencyRef(dep);
 });
 
+alias zip = memoize!(function()
+{
+    Dependency dep;
+    with (dep)
+    {
+        name = "zip";
+        target = "dmdsrc.zip";
+        sources = [thisBuildScript].chain(.sources.root, .sources.backend, .sources.lexer,
+            .sources.frontendHeaders, .sources.dmd).array;
+        msg = "ZIP " ~ target;
+        commandFunction = () {
+            if (exists(target))
+                remove(target);
+            run([env["ZIP"], target.stripExtension] ~ sources);
```

Comment:
Why strip the extension?

Reply:
just copying what the makefiles are doing

Reply:
I've removed the `stripExtension`, well see if it passes the tests